scripts/ci/lib.sh

```diff
@@ -1185,6 +1185,13 @@ store_test_results() {
     local from="$1"
     local to="$2"
 
+    if ! is_in_PR_context; then
+        info "Creating JIRA task for failures found in $from"
+        wget -q https://github.com/janisz/junit2jira/releases/download/v0.0.1/junit2jira && \
+        chmod +x junit2jira && \
+        ./junit2jira -junit-reports-dir "$from"
```

Comment:
Given that it is a prototype have it silently fail and not disrupt following functionality:
```suggestion
{
        info "Creating JIRA task for failures found in $from"
        wget -q https://github.com/janisz/junit2jira/releases/download/v0.0.1/junit2jira && \
        chmod +x junit2jira && \
        ./junit2jira -junit-reports-dir "$from"
} || true
```